agcaballero/portfolio/src/main/java/com/google/sps/servlets/PhotoMapServlet.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.PhotoLocation;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Returns photo location data as a JSON array,
+ */
+@WebServlet("/photo-map-data")
+public class PhotoMapServlet extends HttpServlet {
+  private Collection<PhotoLocation> locations;
+
+  @Override
+  public void init() {
```

Comment:
Is it necessary to call `super.init()`?

Reply:
As far as I know, I don't believe so. I based my code in this section off the similar class (UfoDataServlet) from the examples, which also does not call `super.init()`, and I haven't had issues so far. Would you suggest adding `super.init()`?

Reply:
Ah it looks like super.init() is not necessary when calling the form with no arguments:
https://docs.oracle.com/cd/E19146-01/819-2634/abxbc/index.html

Reply:
Thank you for the info!

Reply:
Done